README.Rmd

```diff
@@ -28,9 +29,19 @@ This package is part of the [hubverse](https://hubverse.io/en/latest/) ecosystem
 
 ## Installation
 
-You can install the development version of hubExamples from [GitHub](https://github.com/) with:
+### Latest
+
+You can install the [latest version of hubExamples from the R-universe](https://hubverse-org.r-universe.dev/hubAdmin):
```

Comment:
should this link be pointing to hubExamples?

Reply:
```suggestion
You can install the [latest version of hubExamples from the R-universe](https://hubverse-org.r-universe.dev/hubExamples):
```